html/templates/combatant.html

```diff
@@ -4,7 +4,7 @@
 <span class="ac" title="Armor Class" data-bind="text: Combatant.AC"></span>
 <span class="tags">
 <!--ko foreach: Combatant.Tags() -->
-    <span class="tag">
+    <span class="tag" data-bind="visible: DurationRemaining() > 0">
```

Comment:
Won't this break tags without durations?

Reply:
Latest commits address this